frontend/pages/sign-up/index.tsx

```diff
@@ -149,178 +203,164 @@ const SignUp: PageComponent<SignUpPageProps, AuthPageLayoutProps> = () => {
-              id="password-description"
-            >
-              <FormattedMessage
-                defaultMessage="Use at least 12 characters, one uppercase letter, one lowercase letter and one number."
-                id="MF4b8Z"
-              />
-            </p>
-            <ErrorMessage id="password-error" errorText={errors.password?.message} />
-          </div>
-          <div className="w-full">
-            <label htmlFor="confirm-password">
-              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
-                <FormattedMessage defaultMessage="Confirm password" id="8HimVK" />
-              </p>
-              <Input
-                type="password"
-                id="confirm-password"
-                name="confirm_password"
-                placeholder={intl.formatMessage({
-                  defaultMessage: 'Insert password',
-                  id: 'HnG9/3',
-                })}
-                aria-describedby="confirm-password-error"
-                register={register}
-              />
-            </label>
-            <ErrorMessage
-              id="confirm-password-error"
-              errorText={errors.confirm_password?.message}
-            />
-          </div>
-        </div>
-        <div className="w-full mt-8">
-          <label htmlFor="accept-terms">
-            <Checkbox
-              name="accept_terms"
-              id="accept-terms"
-              aria-describedby="accept-terms-error"
-              register={register}
-            />
-            <span className="ml-2 font-sans text-sm text-gray-800 font-regular">
-              <FormattedMessage
-                defaultMessage="I agree with the Terms and Privacy Policy."
-                id="X0E3iC"
-              />
-            </span>
-          </label>
-          <ErrorMessage id="accept-terms-error" errorText={errors.accept_terms?.message} />
-        </div>
-        <div className="flex justify-center mt-14">
-          <button
-            className="flex px-5 py-2 font-sans text-sm text-white opacity-75 font-regular rounded-5xl leadign-6 bg-green-dark"
-            type="submit"
-            disabled={signUp.isLoading}
-          >
-            <Loading visible={signUp.isLoading} className="mr-2.5" />
-            <FormattedMessage defaultMessage="Sign up" id="8HJxXG" />
-          </button>
+      {getError(signUp.isError, signUp.error)}
+      {getError(acceptInvitation.isError, acceptInvitation.error)}
+      {!!query.confirmation_token && getError(confirmedUserIsError, confirmedUserError)}
+
+      <div className="flex justify-center">
+        <Loading visible={confirmedUserIsLoading} />
+      </div>
+
+      {!query.confirmation_token && (
+        <div>
+          <p className="mb-1.5 font-sans text-base text-gray-600">
+            <FormattedMessage defaultMessage="Please enter your details below." id="rfVDxL" />
+          </p>
```

Comment:
I don't think you should have moved this paragraph here. It would look better as it was before.
<img alt="The paragraph is shown below the invitation message" src="https://user-images.githubusercontent.com/6073968/195107796-73ea209f-ce6a-403d-8b1d-ee8d0c3ae493.png" height="400" />